src/java_binding/src/lib.rs

```diff
@@ -207,6 +209,16 @@ where
     }
 }
 
+// JNIEXPORT jint JNICALL Java_com_risingwave_java_binding_Binding_vnodeCount
+// (JNIEnv *, jclass);
+
+#[no_mangle]
+pub extern "system" fn Java_com_risingwave_java_binding_Binding_vnodeCount(
+    _env: EnvParam<'_>,
+) -> jint {
+    VirtualNode::COUNT as jint
```

Comment:
Is this a temporary value? I would imagine multiple `Iterator` instances reading the same table, so each iter should read a subset of all vnodes?

Reply:
This is for returning the total number of vnode defined in RisingWave, which is a const value defined in our system.

In the upper level query engine, we will get the total vnode count via this call and assign different subsets of vnodes to different parallelism.